### ISSUE TITLE
Maximum possible investment now a variable

### ISSUE THREAD

**Issue by triceo** [RoboZonky/natural-strategy-setup#30]
Zonky recently introduced a new variable, maximum possible investment - it used to be a constant of 5000 CZK, but now it varies from 5k to 20k based on investor status.

I have implemented this new property here:
https://github.com/RoboZonky/robozonky/commit/32af09b9db74eef1a6cc4d661df044d2f0b97e7c _RoboZonky_ will ask the strategy for maximum investment, compare that against the maximum received from Zonky, and pick whichever number is smaller.

The question now is how to implement this in the web tool, which has no way of knowing what the current user's limit is. My suggestion would be to allow the full range (200 - 20000) and have a warning when going over 5k, informing users that the robot may reduce based on how much they are allowed to invest by Zonky.

**Comment by jhrcek** [comment]
Regarding web tool: allowing 200-20k  all the time is certainly an option, but in order to be more helpful the tool could ask users for their "investor status" (e.g. in the section Výše investice there would be radio button like "Jiz jsem zainvestoval  a) mene nez 100 pujcek b) 101-200 c) 201 nebo vic" with appropriate tooltip about why this is needed) and based on that it would set appropriate maximum investment size. @triceo Wdyt?

**Comment by jhrcek** [comment]
@triceo Implemented according to our discussion.
- It's always legal to set investment size anywhere between 200 and 20k
- When user sets maximum to more than 5k using ANY slider, I display a message like this next to the sliders.

![screenshot from 2017-12-18 17-42-44](https://user-images.githubusercontent.com/2716069/34117191-3c4f05cc-e41b-11e7-9484-9bfdaba4e8ab.png)

Please check it out [here](http://janhrcek.cz/natural-strategy-setup/) and let me know what you think.

 Is the wording and placement of warning ok?

**Comment by triceo** [comment]
Looks good. Three nits to pick:

- In the second paragraph, capitalize "Zonky"
- Right below that, "invastovat".
- I wonder if it's a good idea to list all the specific rules in there. What about just linking to the Zonky website where the rules are listed?

**Comment by triceo** [comment]
Actually, I just noticed... RoboZonky version declaration is missing. I didn't change anything in the UI but the investment size.

![Screenshot](https://user-images.githubusercontent.com/247403/34124969-1b8a404e-e435-11e7-9d9f-c5b837ee8851.png)

**Comment by jhrcek** [comment]
> What about just linking to the Zonky website where the rules are listed?
I agree. Typos fixed and added link to Zonky website instead of listing rules explicitly. Also I removed "než kterou vám Zonky povolí zainvestovat" as I no longer like it in the shorter version.

> version declaration is missing.
Well spotted, I accidentally removed version declaration from the strategy while formatting the code - fixed now. And I bumped the minimum required version to 4.2.0 in the generated strategy.
